.github/workflows/beam_PerformanceTests_Kafka_IO.yml

```diff
@@ -0,0 +1,120 @@
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+      github.event_name == 'schedule' ||
+      github.event.comment.body == 'Run Java KafkaIO Performance Test'
+    runs-on: [self-hosted, ubuntu-20.04, main]
+    timeout-minutes: 120
+    name: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+    strategy:
+      matrix:
+        job_name: ["beam_PerformanceTests_Kafka_IO"]
+        job_phrase: ["Run Java KafkaIO Performance Test"]
+    env:
+      KAFKA_SERVICE_PORT: 32400
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup repository
+        uses: ./.github/actions/setup-action
+        with:
+          comment_phrase: ${{ matrix.job_phrase }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+      - name: Set k8s access
+        uses: ./.github/actions/setup-k8s-access
+        with:
+          cluster_name: io-datastores
+          k8s_namespace: ${{ matrix.job_name }}-${{ github.run_id }}
+      - name: Install Kafka
+        id: install_kafka
+        run: |
+          cd ${{ github.workspace }}/.test-infra/kubernetes/kafka-cluster/
+          kubectl apply -R -f .
+      - name: Set up Kafka brokers
+        id: set_brokers
+        run: |
+          declare -a kafka_service_brokers
+          for INDEX in {0..2}; do
+            kubectl wait svc/outside-${INDEX} --for=jsonpath='{.status.loadBalancer.ingress[0].ip}' --timeout=40m
+            kafka_service_brokers[$INDEX]=$(kubectl get svc outside-${INDEX} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+            echo "KAFKA_SERVICE_BROKER_${INDEX}=${kafka_service_brokers[$INDEX]}" >> $GITHUB_OUTPUT
+          done
+      - name: Prepare test arguments
+        uses: ./.github/actions/test-arguments-action
+        with:
+          test-type: performance
+          test-language: java
+          argument-file-paths: |
+            ${{ github.workspace }}/.github/workflows/performance-tests-job-configs/kafka_IO_Streaming.txt
+            ${{ github.workspace }}/.github/workflows/performance-tests-job-configs/kafka_IO_Batch.txt
+          arguments: |
+            --kafkaBootstrapServerAddresses=${{ steps.set_brokers.outputs.KAFKA_SERVICE_BROKER_0 }}:${{ env.KAFKA_SERVICE_PORT }},${{ steps.set_brokers.outputs.KAFKA_SERVICE_BROKER_1 }}:${{ env.KAFKA_SERVICE_PORT }},${{ steps.set_brokers.outputs.KAFKA_SERVICE_BROKER_2 }}:${{ env.KAFKA_SERVICE_PORT }}
+      - name: Run Java KafkaIO Performance Streaming Test
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:java:io:kafka:integrationTest
+          arguments: |
+            --tests org.apache.beam.sdk.io.kafka.KafkaIOIT.testKafkaIOReadsAndWritesCorrectlyInStreaming \
+            --info \
+            -DintegrationTestRunner=dataflow \
+            -DintegrationTestPipelineOptions='[${{env.beam_PerformanceTests_Kafka_IO_test_arguments_1}}]' \
```

Comment:
Please declare this variable above in the env section so that the linter does not display a warning. Please do this for all workflow files in the PR.